microsoft-edge/extensions/publish/publish-extension.md

```diff
@@ -54,11 +54,11 @@ Before uploading your package to Partner Center, inspect the manifest file to ma
 
 The following fields in the manifest file are automatically included in your extension's product listing at Microsoft Edge Add-ons.  The fields are read-only on the **Store Listings** page, which is described below.
 
-*  The `Name` field, which populates the **Display name** on the store details page.
+*  The `Name` field, which populates the **Extension name** on the **Details for \<Language\>** page.
 
-*  The `Description` field, which populates the **Short description** on the store details page.
+*  The `Description` field, which populates the **Description** on the **Details for \<Language\>** page.
```

Comment:
tech review this change from **Short description** to **Description**

@G-u-r-j-e-e-t 

rendered:

present section:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions/publish/publish-extension?branch=pr-en-us-3645#review-manifest-field-values

UI screenshot section shows only a **Description** field, not a **Short description** field:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions/publish/publish-extension?branch=pr-en-us-3645#step-5-add-store-listing-details-for-the-extension
direct link to png: 
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions/publish/publish-extension-images/details-page.png